app/component/summary/customize-search.cjsx

```diff
@@ -64,85 +64,95 @@ class CustomizeSearch extends React.Component
         </div>
       </section>
 
-      <section className="offcanvas-section">
-        <Slider
-          headerText={@context.intl.formatMessage(
-            {id: 'walking', defaultMessage: "Walking"})}
-          defaultValue={@context.getStore('ItinerarySearchStore').getWalkReluctance()}
-          onSliderChange={(e) => @context.executeAction(ItinerarySearchAction.setWalkReluctance, e.target.value)}
-          min={0.8}
-          max={10}
-          step={0.2}
-          minText={@context.intl.formatMessage(
-            {id: 'prefer-walking', defaultMessage: "Prefer walking"})}
-          maxText={@context.intl.formatMessage(
-            {id: 'avoid-walking', defaultMessage: "Avoid walking"})}
-        />
-      </section>
-      <section className="offcanvas-section">
-        <Slider
-          headerText={@context.intl.formatMessage(
-            {id: 'transfers', defaultMessage: "Transfers"})}
-          defaultValue={@context.getStore('ItinerarySearchStore').getWalkBoardCost()}
-          onSliderChange={(e) => @context.executeAction(ItinerarySearchAction.setWalkBoardCost, e.target.value)}
-          min={1}
-          max={1800}
-          step={60}
-          minText={@context.intl.formatMessage(
-            {id: 'transfers-allowed', defaultMessage: "Transfers allowed"})}
-          maxText={@context.intl.formatMessage(
-            {id: 'avoid-transfers', defaultMessage: "Avoid transfers"})}
-        />
-      </section>
-      <section className="offcanvas-section">
-        <Slider
-          headerText={@context.intl.formatMessage(
-            {id: 'transfers-margin', defaultMessage: "Transfer margin"})}
-          defaultValue={@context.getStore('ItinerarySearchStore').getMinTransferTime()}
-          onSliderChange={(e) => @context.executeAction(ItinerarySearchAction.setMinTransferTime, e.target.value)}
-          min={60}
-          max={660}
-          step={30}
-          minText={"1 min"}
-          maxText={"12 min"}
-        />
-      </section>
-      <section className="offcanvas-section">
-        <Slider
-          headerText={@context.intl.formatMessage(
-            {id: 'walking-speed', defaultMessage: "Walking speed"})}
-          defaultValue={@context.getStore('ItinerarySearchStore').getWalkSpeed()}
-          onSliderChange={(e) => @context.executeAction(ItinerarySearchAction.setWalkSpeed, e.target.value)}
-          min={0.5}
-          max={3}
-          step={0.1}
-          minText={@context.intl.formatMessage(
-            {id: 'slow', defaultMessage: "Slow"})}
-          maxText={@context.intl.formatMessage(
-            {id: 'run', defaultMessage: "Run"})}
-        />
-      </section>
+      {if config.customizeSearch.walkReluctance.available
```

Comment:
Maybe. If we had more of the variables configured, we could loop through the configured sliders and create them dynamically. For now I think this is viable.